synapse/axon.py

```diff
@@ -147,9 +151,18 @@ class Axon(s_cell.Cell):
         self.axonmetrics.setdefault('size:bytes', 0)
         self.axonmetrics.setdefault('file:count', 0)
 
+        self.on('syn:health', self._onHealthAxon)
```

Comment:
Any reason we're using events here rather than just methods and sub-class methods?

Reply:
this was discussed out of band; the reason for events was to allow a cell to have multiple subscribers to the event and report up without having to implement a function over/callback mechanism.  the first consumer of this could be core modules.